solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -374,14 +403,33 @@ contract RandomBeacon is Ownable {
         );
     }
 
-    /// @notice Registers caller in the sortition pool.
-    function registerMemberCandidate() external {
+    /// @notice Registers the caller in the sortition pool.
+    /// @dev Creates a gas deposit tied to the operator address. The gas
+    ///      deposit is released when the operator is banned for sortition pool
```

Comment:
I think it should be _banned from_.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2707/commits/c97dba585f021ce3e8ec7fb37e2665a4f7577541 as part of #2707